.github/workflows/test-coverage.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Google LLC
```

Comment:
what are these files?

Reply:
These .github files haven't been added by me. These aren't actual changes, the current state of the Harbourbridge repository already has these files with the same content in them.